packages/@sanity/client/test/client.test.js

```diff
@@ -363,6 +363,43 @@ test('can query for single document', t => {
     .then(t.end)
 })
 
+test('can query for multiple documents', t => {
+  nock(projectHost())
+    .get('/v1/data/doc/foo/abc123,abc321')
+    .reply(200, {
+      ms: 123,
+      documents: [{_id: 'abc123', mood: 'lax'}, {_id: 'abc321', mood: 'tense'}]
+    })
+
+  getClient()
+    .getDocuments(['abc123', 'abc321'])
+    .then(([abc123, abc321]) => {
+      t.equal(abc123.mood, 'lax', 'data should match')
+      t.equal(abc321.mood, 'tense', 'data should match')
+    })
+    .catch(t.ifError)
+    .then(t.end)
+})
+
+
+test('preserves the position of requested documents', t => {
+  nock(projectHost())
+    .get('/v1/data/doc/foo/abc123,abc321')
+    .reply(200, {
+      ms: 123,
+      documents: [{_id: 'abc321', mood: 'tense'}]
+    })
+
+  getClient()
+    .getDocuments(['abc123', 'abc321'])
```

Comment:
Perhaps add a third document (which also exist, but is returned in the wrong order), so as to make sure we treat both missing and present documents correctly?